content/en/docs/ops/configuration/traffic-management/dns-proxy/index.md

```diff
@@ -219,6 +223,64 @@ A virtual IP address will be assigned to every service entry so that client side
     ADDRESS=240.240.69.138, DESTINATION=Cluster: outbound|9000||tcp-echo.external-1.svc.cluster.local
     {{< /text >}}
 
+## DNS Auto Allocation V2
+
+Istio now offers an enhanced implementation of DNS Auto Allocation. To use the new feature, replace the feature flag `ISTIO_META_DNS_AUTO_ALLOCATE`, which was used in the previous example, with `PILOT_ENABLE_IP_AUTOALLOCATE` while installing Istio. All examples given so far would work as is.
```

Comment:
This is a bit strange because the enablement also moves from meshConfig to a pilot environment variable. Should we mention that somehow? A quick reading of this without looking at your yaml might assume it is still in meshConfig

Reply:
Sure, let me clarify this part